testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/AbstractConsoleTest.java

```diff
@@ -77,25 +80,35 @@ public void beforeConsoleTest() {
     }
 
     public void loginToMasterRealmAdminConsoleAs(UserRepresentation user) {
-        loginToAdminConsoleAs(adminConsolePage, loginPage, user);
+        loginToAdminConsoleAs(adminConsolePage, loginPage, user, false);
```

Comment:
I don't understand the point in adding this. This is not the correct place to test remember me.